tools/staramr/staramr_search.xml

```diff
@@ -17,7 +21,7 @@
             #set $named_genomes = $named_genomes + ' ' + $_named_genome
         #end for
 
-        #set $excel_proper_extension = '"results.xlsx"'
+        #set $excel_proper_extension = 'results.xlsx'
         ln -s "$excel" $excel_proper_extension &&
```

Comment:
I think you want to use single quotes over double quotes `"$excel"`:

https://github.com/galaxyproject/tools-iuc/pull/3893#discussion_r693192671

Reply:
Done in 03d216e38e96f0e1b418f13b3424d9d7aec4db40